### DISCUSSION TITLE
Do we need to restart flowise everytime we update custom tool?

### DISCUSSION THREAD

**Discussion by VincentEmmanuel** [FlowiseAI/Flowise#2231] [Q&A] [answered]
Hi everyone, 

I noticed that after updating custom tools and going into the flow, it sort of retained the old cutom tool information. If we restart flowise, then it will be "refreshed". 

Is there any other way to refresh it other than restarting flowise?

**Comment by acdewinter** [comment, marked as answer]
I found the same problem, you can overcome this by "deleting" the selected custom tool on the node itself, and re-selecting it from the drop down and then hitting "save" on your flow. Initially I was deleting the whole node and re-creating it, but that is laborious and so far this is the fastest method around the problem.

**Reply by VincentEmmanuel** [reply]
Thanks let me try it out!
